src/main/java/taxi/model/Driver.java

```diff
@@ -40,21 +60,12 @@ public void setLicenseNumber(String licenseNumber) {
     }
 
     @Override
-    public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
-        Driver driver = (Driver) o;
-        return Objects.equals(id, driver.id)
-                && Objects.equals(name, driver.name)
-                && Objects.equals(licenseNumber, driver.licenseNumber);
+    public boolean equals(Object obj) {
+        return super.equals(obj);
```

Comment:
why do we need to change equals logic?

Reply:
because we added to Driver two more fields? Remade implementation...